### PR TITLE
Full Polish translation

### PR DESCRIPTION
**Pull request by ajes**
All files were translated for the Polish language.

### CODE REVIEW

File: store/store-description.pl.bbcode [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
+Robo Instructus to gra logiczna, w której gracze sterują robotem, wydając instrukcje za pomocą prostego języka programowania.
 
-As players progress through the game they unlock new functions to overcome new puzzles, each of which can be solved in multiple ways. The more you master the robot, the more elegant and powerful your solutions will be.
+W miarę postępów w grze odblokowujesz nowe funkcje, aby pokonać nowe zagadki, z których każda może być rozwiązana na wiele sposobów. Im lepiej opanujesz robota, tym bardziej eleganckie i skuteczne będą twoje rozwiązania.
 
-Take the role of a Salvage Engineer sent across space. Use wits and tenacity to uncover the secrets of this isolated, frozen world.
+Przyjmij rolę Inżyniera ratunkowego wysłanego w kosmos. Wykorzystaj swoją inteligencję i determinację, aby odkryć tajemnice tego izolowanego, zamrożonego świata.
 
-Note: Game text translation incomplete. The game includes untranslated English text. https://github.com/big-ab-games/robo-instructus-translation#about
+Uwaga: Tłumaczenie tekstu gry jest niekompletne. Gra zawiera nieprzetłumaczony tekst w języku angielskim. https://github.com/big-ab-games/robo-instructus-translation#about
```
Comment (alexheretic) [inline store/store-description.pl.bbcode:9]: Presumably we can just drop this line now the text _will_ be fully translated?

**Comment by alexheretic** [comment]
Looks great thanks a lot!

# Primer screenshot
primer pl

# Colony screenshot
colony pl

File: store/store-description.pl.bbcode (hunk 1 above)
Comment (alexheretic) [reply store/store-description.pl.bbcode:9]: Actually I guess I still have it for de etc, maybe because there are still odd bits in english

**Comment by ajes** [comment]
Yep, looks nice. As I am playing game right now, when you will push update with polish lang, I will look more carefully how it looks in the context of playing. Polish language is sometimes nightmare when you do translations, as context matters a lot.

**Pull request merged**